File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string str = to_string(x);
        int st=0;int end=str.length()-1;
        while(st<=end){
            if(str[st]!=str[end]){
                return false;
            }
            st++;
            end--;
        }
        return true;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        s = str(x)
        st=0
        end=len(s)-1
        while st<end:
            if s[st]!=s[end]:
                return False
            st +=1
            end-=1

        return True